src/org-manager/containers/org-pending-approve/org-pending-approve.component.ts

```diff
@@ -27,6 +28,9 @@ export class OrgPendingApproveComponent implements OnInit, OnDestroy {
                     this.onGoBack();
                 }
             });
+        this.store.pipe( select(fromOrganisationPendingStore.pendingOrganisations)).subscribe(something => {
+           console.log(something['responseMessages']);
```

Comment:
create selector that returns the error message from the state and hook it to html